db/schema.rb

```diff
@@ -348,15 +348,6 @@
     t.index ["service_provider_id"], name: "index_integrations_on_service_provider_id"
   end
 
-  create_table "irs_attempt_api_log_files", force: :cascade do |t|
```

Comment:
I'm not sure about this. Should we keep or remove it?

Reply:
I say remove it